proxy/logging/Log.cc

```diff
@@ -1242,7 +1242,15 @@ Log::flush_thread_main(void * /* args ATS_UNUSED */)
           break;
         }
 
+        int logfilefd = logfile->get_fd();
```

Comment:
I tweaked this to be moved out of the loop. If it's OK when the loop starts it will be OK every loop iteration. And in that case, we might as well depend on `check_fd` to have verified that and just use an `ink_assert` to make Coverity happy.